src/Language/Haskell/Brittany/Internal/Layouters/Stmt.hs

```diff
@@ -55,38 +55,63 @@ layoutStmt lstmt@(L _ stmt) = do
                                 -- heh.
       Just []        -> docLit $ Text.pack "let" -- this probably never happens
       Just [bindDoc] -> docAltFilter
-        [ ( indentPolicy /= IndentPolicyLeft
+        [ -- let bind = expr
+          ( indentPolicy /= IndentPolicyLeft
```

Comment:
Is there a simple way to enable this when the expression is actually a single line? That might be too much complexity thought :smile:

Reply:
Like this? I think this works.. oh I broke the tests.

Reply:
But they highlight the changes. And looks approximately correct (although the test coverage is not to be trusted entirely.)